src/main/java/org/opensearch/security/support/ModuleType.java

```diff
@@ -52,7 +52,7 @@
 	REST_MANAGEMENT_API("REST Management API", "org.opensearch.security.dlic.rest.api.SecurityRestApiActions", Boolean.TRUE),
 	DLSFLS("Document- and Field-Level Security", "org.opensearch.security.configuration.SecurityFlsDlsIndexSearcherWrapper", Boolean.TRUE),
 	AUDITLOG("Audit Logging", "org.opensearch.security.auditlog.impl.AuditLogImpl", Boolean.TRUE),
-	MULTITENANCY("Kibana Multitenancy", "org.opensearch.security.configuration.PrivilegesInterceptorImpl", Boolean.TRUE),
+	MULTITENANCY("OpenSearch Dashboards  Multi-tenancy", "org.opensearch.security.configuration.PrivilegesInterceptorImpl", Boolean.TRUE),
```

Comment:
nit: extra space between Dashboards and Multi-tenancy

Reply:
fixed